flashlib.frc.nt/src/main/java/com/flash3388/frc/nt/combined/NtTable.java

```diff
@@ -0,0 +1,47 @@
+package com.flash3388.frc.nt.combined;
+
+import edu.wpi.first.networktables.*;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class NtTable {
+
+    private final NetworkTable mTable;
+
+    public NtTable(String name) {
+        this(NetworkTableInstance.getDefault(), name);
+    }
+
+    public NtTable(NetworkTableInstance tableInstance, String name) {
+         this(tableInstance.getTable(name));
+    }
+
+    public NtTable(NetworkTable table) {
```

Comment:
Ctor convention in Flashlib stuff is opposite: general contructor first, overloads bellow.

Reply:
BUT WHY?

Reply:
Because I said so